pennylane_lightning/src/algorithms/AdjointDiff.hpp

```diff
@@ -490,6 +513,100 @@ template <class T = double> class AdjointJacobian {
         }
     }
 
+    /**
+     * @brief OpenMP accelerated application of observables to given
+     * statevectors
+     *
+     * @param states Vector of statevector copies, one per observable.
+     * @param reference_state Reference statevector
+     * @param observables Vector of observables to apply to each statevector.
+     */
+    inline void applyObservables(std::vector<StateVectorManaged<T>> &states,
+                                 const StateVectorManaged<T> &reference_state,
+                                 const std::vector<ObsDatum<T>> &observables) {
+        // Globally scoped exception value to be captured within OpenMP block.
+        // See the following for OpenMP design decisions:
+        // https://www.openmp.org/wp-content/uploads/openmp-examples-4.5.0.pdf
+        std::exception_ptr ex = nullptr;
+        size_t num_observables = observables.size();
+#if defined _OPENMP
+#pragma omp parallel default(none)                                             \
+    shared(states, reference_state, observables, ex, num_observables)
+        {
```

Comment:
Does this need an indent?
(I know indentation doesn't matter in C++, but more so for readability.)

Reply:
As above, I disabled the formatter here and added appropriate indentation.